### DISCUSSION TITLE
Wiki? Manual?

### DISCUSSION THREAD

**Discussion by TheHamiltonian** [epochpic/epoch#170] [Q&A] [answered]
Hi,

is the Wiki site inaccessible? Where can we find updated PDF containing the manual?

**Comment by TomGoffrey** [comment, marked as answer]
You can see the latest version of the user manual in the 4.14 release: https://github.com/Warwick-Plasma/epoch/releases/tag/v4.17.14

The information on the wiki will be made accessible again soon, although it's likely it'll be in a different format.
